.github/workflows/update-as.yml

```diff
@@ -29,11 +29,12 @@ jobs:
     - name: Get new A-S tag to be used in the PR info
       run: |
         cd test-fixtures/
+        chmod u+x read_tag_test.sh
```

Comment:
Not entirely sure if this will work for the permission problem.

Reply:
I changed the permission of the file locally and upload it again. Hopefully this will be solved...